ray-operator/DEVELOPMENT.md

```diff
@@ -59,7 +60,6 @@ IMG=kuberay/operator:nightly make docker-build
 # Command: kind load docker-image {IMG_REPO}:{IMG_TAG}
 kind load docker-image kuberay/operator:nightly
 
```

Comment:
Do you want to load nightly? It might not have all your current development. I think its better to build the image locally and then load it. See how API server make does it.

Reply:
While the image reference is `kuberay/operator:nightly`, it's actually built from the local source code in the previous step, and then loaded into KinD. So despite the reference being a bit confusing, I think it does take current development into account.